### PR TITLE
docs: update content to angualr 19

### PR DESCRIPTION
**Pull request by Theeraphat-Sorasetsakul**
## Description

Update content in Angular page to Angular 19

Fixes # ([DME-37816](https://jira.refinitiv.com/browse/DME-37816))

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by dtanp-rft** [comment]
We should update this page too https://ui.refinitiv.com/guides/compatibility

File: documents/src/pages/build-app/framework-integration/angular.md [hunk 1]
```diff
@@ -12,13 +12,35 @@ layout: default
...
+```diff
+-import { Component } from '@angular/core';
++import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+ import { Component } from '@angular/core';
+
+ @Component({
+   selector: 'app-root',
+   templateUrl: './app.component.html',
+-  styleUrl: './app.component.css'
++  styleUrl: './app.component.css',
```
Comment (Sakchai-Refinitiv) [inline documents/src/pages/build-app/framework-integration/angular.md]: The styleUrl hasn't changed

File: documents/src/pages/build-app/framework-integration/angular.md [hunk 2]
```diff
@@ -27,14 +49,16 @@ Make the following changes to `./src/app/app.module.ts`.
...
    declarations: [
      AppComponent
    ],
    imports: [
-     BrowserModule
+     BrowserModule,
+     AppRoutingModule
    ],
    providers: [],
 -  bootstrap: [AppComponent]
```
Comment (Sakchai-Refinitiv) [inline documents/src/pages/build-app/framework-integration/angular.md]: The bootstrap hasn't changed

Comment (Theeraphat-Sorasetsakul) [reply documents/src/pages/build-app/framework-integration/angular.md]: Removed.

File: documents/src/pages/build-app/framework-integration/angular.md (hunk 1 above)
Comment (Theeraphat-Sorasetsakul) [reply documents/src/pages/build-app/framework-integration/angular.md]: Removed.

**Review by Sakchai-Refinitiv — APPROVED** [review]
LGTM

**Pull request merged**